keyboards/planck/keymaps/solstice22/config.h

```diff
@@ -0,0 +1,96 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+#define ONE_UP_SOUND \
```

Comment:
Most of these songs are in the `song_list.h` files.  So you shouldn't need them, as all, and should be removed.